### PR TITLE
Added GitHub Ribbon

### LINKED ISSUE #89: add github ribbon with link to this repo :)
https://github.com/tholman/github-corners something like this in rob right, maybe with some opacity

### PR DESCRIPTION
**Pull request by amd-9**
Added GitHub Ribbon bage in top right corner

image

Closes: #89

### CODE REVIEW

File: client/src/GitHubRibbon.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { ReactComponent as GitHubRibbonIcon } from './assets/github-ribbon.svg'
+
+function GitHubRibbon() {
+  return (
+    <a
+      href="https://github.com/tarnas14/ti4-companion"
+      className="github-corner"
+      aria-label="View source on GitHub"
+    >
+      <GitHubRibbonIcon onClick={() => console.log('Clicked!')} />
```
Comment (tarnas14) [inline client/src/GitHubRibbon.js]: plrease remove the onClick console.log listener :)

File: client/src/index.css [hunk 2]
```diff
@@ -34,3 +34,32 @@ body, *:fullscreen {
   background-color: transparent !important;
   box-shadow: none !important;
 }
+
+/* GitHub Ribbon*/
+.github-corner:hover .octo-arm {
```
Comment (tarnas14) [inline client/src/index.css:43]: please make the ribbon semi-transparent (maybe opacity .8 ?) by default and only be opacity: 1 when you `:hover` over it

(we don't want too much focus of our users on this)

**Review by tarnas14 — CHANGES_REQUESTED** [review]
thank you for your contribution!

looking great, I have 2 small comments, though :)

**Comment by amd-9** [comment]
He he, forgot about output via console.log :) I've removed console.log and added opacity for github ribbon

File: client/src/GitHubRibbon.js (hunk 1 above)
Comment (amd-9) [reply client/src/GitHubRibbon.js]: Yep. Removed onClick

File: client/src/index.css (hunk 2 above)
Comment (amd-9) [reply client/src/index.css:43]: Added opatcity for github ribbon

**Comment by tarnas14** [comment]
@amd-9 umm... are you sure you pushed the changes? :) you answered the comments, but no new commits have appeared

**Comment by amd-9** [comment]
@tarnas14 my apologize :) seems like at the end of the day I've just forgot to push

**Review by tarnas14 — CHANGES_REQUESTED** [review]
functionality is great, please remove package-lock.json, though, we are using yarn.lock here

for that matter - could you add `package-lock.json` to root `.gitignore` file so that this doesn't happen any more? :)

File: client/.gitignore [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+# package-lock.json - since yarn is used
+package-lock.json
```
Comment (tarnas14) [inline client/.gitignore:4]: great, however the `client/package-lock.json` was still committed in the history

you'll have to run `git rm --cached package-lock.json` and commit that change to remove it from git history, I'm afraid

Comment (amd-9) [reply client/.gitignore:4]: Thanks! I've removed package-lock.json from PR

**Review by tarnas14 — APPROVED** [review]
perfect

will merge once the linter checks go through

thank you for your contribution! :)

**Pull request merged**